### PR TITLE
Adds additional JS to System.Web.Extensions

### PR DESCRIPTION
**Pull request by steveisok**
Fixes https://github.com/microsoft/referencesource/issues/121

# READ BEFORE FILING

This repository is read-only and doesn't accept community pull requests.

### CODE REVIEW

File: System.Web.Extensions/Script/ExtensionsScript.csproj.vspscc
Comment (danmosemsft) [inline System.Web.Extensions/Script/ExtensionsScript.csproj.vspscc:1]: Please remove this file. . it's a source control artefact

File: System.Web.Extensions/Script/ExtensionsScript.csproj
Comment (danmosemsft) [inline System.Web.Extensions/Script/ExtensionsScript.csproj:1]: Please also remove this -- it's not useful out side of our build system.

**Review by danmosemsft — APPROVED** [review]
We should also get @michaelketting to confirm it's complete before we merge.

**Comment by MichaelKetting** [comment]
@danmosemsft, @steveisok  Thank you! I will verify the list sometime in the next couple of workdays.

**Comment by MichaelKetting** [comment]
@danmosemsft, do you happen to have a link to a publicly available copy of Nikhil Kothari's JS Utils? Or, would it be possible (i.e. legally, license-wise) to publish the copy you are using? As far as I can tell, that was the tool used to process/compile the JS and JSA files into their output format? Nikhil unfortunately removed them during a revamp of his homepage it was never published on Codeplex or Github as far as I can tell.

Thanks, Michael!

**Comment by danmosemsft** [comment]
@michaelketting I'm not from that team so I don't know and likely @steveisok does not either. 

Maybe @tratcher might know.

I don't think this should hold up merging this PR, right?

**Comment by MichaelKetting** [comment]
@danmosemsft Thank you for pinging the correct person. Regarding the merge: yes and no. we want to check that all the files are there, we'll now do this manually via some grep or so :) instead of just running the preprocesser/compiler and checking the entire output for equality against the .NET Framework embedded resources. We'll do this early in the upcoming week.

**Review by MichaelKetting — APPROVED** [review]
@steveisok okay, we've now checked the files and all looks good from our side. Thank you!

**Review by StephenMolloy — APPROVED** [review]
PR looks good to me. I don't see any missing script files, and the ones included here match the sources in TFS.

**Pull request merged**
